packages/chains/src/bsc/assets.ts

```diff
@@ -1,8 +1,18 @@
 import { assetSymbols, OracleTypes, SupportedAsset } from "@midas-capital/types";
 
-import { ankrBNBDocs, defaultDocs, ellipsisDocs, jarvisDocs, pancakeSwapDocs, stkBNBDocs } from "../common";
+import {
+  ankrBNBDocs,
+  apeSwapDocs,
+  BNBxDocs,
+  defaultDocs,
+  ellipsisDocs,
+  jarvisDocs,
+  pancakeSwapDocs,
+  stkBNBDocs,
+} from "../common";
 
 export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const BNB = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
lol? Is it a real token? Looks like an EOA to me.
So native token? Maybe use ZeroAddress again?

Reply:
That's the address for native BNB on BSC. See: https://ellipsis.finance/pool/0xFD4afeAc39DA03a05f61844095A75c4fB7D766DA check the BNB token. It is the null address